generate-routes.ts

```diff
@@ -206,27 +176,27 @@ const deriveResource = (
   endpointPath: string,
   method: Method,
 ): string | null => {
-  if (isEndpointResource(endpointPath)) {
-    return endpointResources[endpointPath] ?? null
-  }
-
   if (isOpenapiPath(endpointPath)) {
     const spec = openapi.paths[endpointPath]
     const methodKey = method.toLowerCase()
 
     if (methodKey === 'post' && 'post' in spec) {
-      const response = spec.post.responses[200]
-      if (!('content' in response)) return null
-      return deriveResourceFromSchema(
```

Comment:
Can we remove deriveResourceFromSchema now?

Reply:
Sure! Like this https://github.com/seamapi/javascript-http/pull/87/commits/bbb5d9ae73e06b8e019ffcc54777f4ee3f1f495c ?

Reply:
Slight miscommunication, I didn't mean to suggest we should remove the function and inline the logic, I really was just asking if we still need that logic. Why can't we use `openapiResponseKeyProp`?

Maybe a better question is, should we keep `deriveResourceFromSchema` but replace the logic to use `openapiResponseKeyProp`?

Reply:
Oh, I got it now! I think we attach the openapiResponseKeyProp header only to POST schemas, which means currently it can't be used for other methods. You can check out the source code [here](https://github.com/seamapi/nextlove/blob/8f83afb015a398a096e3a6248bc641e4b81ae737/packages/nextlove/src/generators/generate-openapi/fern-sdk-utils.ts#L68).

Reply:
Ok, let's keep the `deriveResourceFromSchema` function then, but we can rename it to `deriveResourceFromSchemaForGetRequest` and use it only for POST. 

We can note in this issue that we can remove that function in the future: https://github.com/seamapi/javascript-http/issues/91

Reply:
https://github.com/seamapi/javascript-http/pull/87/commits/e61b1cd74912486723c506a8b4190705529989a5